src/stores/keybindingStore.ts

```diff
@@ -51,30 +50,19 @@ export class KeyComboImpl implements KeyCombo {
     })
   }
 
-  equals(other: any): boolean {
-    if (toRaw(other) instanceof KeyComboImpl) {
-      return (
-        this.key === other.key &&
-        this.ctrl === other.ctrl &&
-        this.alt === other.alt &&
-        this.shift === other.shift
-      )
-    }
-    return false
-  }
+  equals(other: unknown): boolean {
+    const raw = toRaw(other)
 
-  serialize(): string {
-    return `${this.key}:${this.ctrl}:${this.alt}:${this.shift}`
+    return raw instanceof KeyComboImpl
+      ? this.key.toUpperCase() === raw.key.toUpperCase() &&
```

Comment:
Special note: this line keeps backward compatibility with existing keybindings user saved. The existing keybindings will not be migrated to upper case format.